repositories/user.go

```diff
@@ -0,0 +1,70 @@
+package repositories
+
+import (
+	"database/sql"
+
+	"github.com/ShinnosukeSuzuki/techtrain-mission-ca-tech-dojo-golang/models"
+)
+
+// リポジトリ構造体を定義
+type UserRepository struct {
+	db *sql.DB
+}
+
+// リポジトリのコンストラクタ
+func NewUserRepository(db *sql.DB) UserRepository {
+	return UserRepository{db: db}
+}
+
+// nameとtokenから新規ユーザーを作成する
+func (r *UserRepository) CreateUser(name string, token string) (models.User, error) {
+	const sqlInsertUser = `
+		INSERT INTO users (name, token)
+		VALUES (?, ?);
+	`
+
+	var newUser models.User
+	newUser.Name = name
+	newUser.Token = token
+	result, err := r.db.Exec(sqlInsertUser, newUser.Name, newUser.Token)
```

Comment:
現状Modelのバリデーションを実装しているわけではないと思うので、一度Modelに変換する必要はなさそうに見えますね。

```suggestion
	result, err := r.db.Exec(sqlInsertUser, name, token)
```


もし、 `models.NewUser(name, token string)` みたいなモデルを生成するメソッドを作成して、その中でドメイン制約（バリデーション）を加えているような実装の場合は有効だと思います！

Reply:
https://github.com/ShinnosukeSuzuki/techtrain-mission-ca-tech-dojo-golang/pull/1/commits/aaa64c909b39716844c5882755c82fb691dbcad4
上記コミットで修正しました。